talktorials/11_online_apis/11c_quality_assessment.ipynb

```diff
@@ -0,0 +1,1370 @@
+    "- Ferreira de Freitasa, R.; Schapira, M. Med. Chem. Commun., 2017,8, 1970-1981. DOI: 10.1039/C7MD00381A\n",
+    "- Klebe G. (2013) Protein–Ligand Interactions as the Basis for Drug Action. In: Klebe G. (eds) Drug Design. Springer, Berlin, Heidelberg. DOI: 10.1007/978-3-642-17907-5_4\n",
+    "- Salentin,S. et al. PLIP: fully automated protein-ligand interaction profiler. Nucl. Acids Res. (1 July 2015) 43 (W1): W443-W447. doi: 10.1093/nar/gkv315\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Theory\n",
+    "\n",
+    "### Protein-ligand interactions\n",
+    "\n",
+    "Ligand binding is mainly governed by noncovalent interactions between the ligand and the surface of the protein pocket or protein-protein interface. This process is a function of electrostatic and shape complementarities, induced fitting, desolvation processes and more.\n",
+    "\n",
+    "Some quotes from the literature:\n",
+    "\n",
+    "Adapted from [José L. Medina-Franco, Oscar Méndez-Lucio, Karina Martinez-Mayorga](https://www.sciencedirect.com/science/article/pii/S1876162314000029):\n",
+    "\n",
+    "> Understanding protein–ligand interactions (PLIs) and protein–protein interactions (PPIs) is at the core of molecular recognition and has a fundamental role in many scientific areas. PLIs and PPIs have a broad area of practical applications in drug discovery including but not limited to molecular docking, structure-based design, virtual screening of molecular fragments, small molecules, and other type of compounds, clustering of complexes, and structural interpretation of activity cliffs, to name a few.\n",
+    "\n",
+    "Of course, these interactions can be rationalized in several ways, which opens the door to systematic analysis of the docking solutions. For example, as adapted from [Med. Chem. Commun., 2017,8, 1970-1981](https://pubs.rsc.org/en/content/articlelanding/2017/md/c7md00381a#!divAbstract):\n",
+    "\n",
+    "> We extracted from the PDB all X-ray structures of small-molecules in complex with proteins, with a resolution ≤2.5 Å, resulting in a collection of 11 016 complexes. To be considered as a ligand, the compound had to meet several criteria such as being a small molecule and be of interest for medicinal chemistry applications (buffers or part of crystallization cocktails were excluded. See ESI† for more details). This collection contained 750 873 ligand–protein atom pairs, where a pair of atoms is defined as two atoms separated by 4 Å or less. The top-100 most frequent ligand–protein atom pairs can be clustered into seven interaction types (see figure below). Among the most frequently observed are interactions that are well known and widely used in ligand design such as hydrophobic contacts, hydrogen bonds and π-stacking. These are followed by weak hydrogen bonds, salt bridges, amide stacking, and cation–π interactions."
```

Comment:
To avoid splitting numbers over two lines, maybe use thousands separators?
```suggestion
    "> We extracted from the PDB all X-ray structures of small-molecules in complex with proteins, with a resolution ≤2.5 Å, resulting in a collection of 11,016 complexes. To be considered as a ligand, the compound had to meet several criteria such as being a small molecule and be of interest for medicinal chemistry applications (buffers or part of crystallization cocktails were excluded. See ESI† for more details). This collection contained 750,873 ligand–protein atom pairs, where a pair of atoms is defined as two atoms separated by 4 Å or less. The top-100 most frequent ligand–protein atom pairs can be clustered into seven interaction types (see figure below). Among the most frequently observed are interactions that are well known and widely used in ligand design such as hydrophobic contacts, hydrogen bonds and π-stacking. These are followed by weak hydrogen bonds, salt bridges, amide stacking, and cation–π interactions."
```